ocaml/utils/clflags.mli

```diff
@@ -59,6 +59,9 @@ module Libloc : sig
   }
 end
 
+type profile_column = [ `Time | `Alloc | `Top_heap | `Abs_top_heap | `Counters ]
```

Comment:
Why did `column` move here from `Profile` ?

Reply:
`Profile` needed to know what command line arguments were set to determine whether to compute the counter function or not. So, `column` had to move here to avoid creating a circular dependency between `Profile` and `Clflags`.